### PR TITLE
Accept named values in units() fn

### PR DESCRIPTION
**Pull request by thisisdano**
- Now the `units()` function can accept the USWDS named spacing values, like `units(card)` or units(`mobile-lg`)`
- Rewrite `u-padding()` mixin to use `get-uswds-value()` not `units()` fn

### CODE REVIEW

**Comment by maya** [comment]
Will `units(1)` still work? Bc I remember using that on the doc site so will want to update it as well.

**Comment by thisisdano** [comment]
Yeah, any numeric value should still work on `units()` — like `units(1)` or `units(.5)`

File: src/stylesheets/core/mixins/utilities/_padding.scss [hunk 1]
```diff
@@ -8,18 +8,18 @@
     $important: ' !important';
   }
   @if $side == all {
-    padding: units($value...)#{$important};
+    padding: get-uswds-value(padding, $value...)#{$important};
```
Comment (el-mapache) [inline src/stylesheets/core/mixins/utilities/_padding.scss]: This seems fine, but I'm not sure I understand why this change was made. Can you give me a little context?

Comment (thisisdano) [reply src/stylesheets/core/mixins/utilities/_padding.scss]: Mostly to make it consistent with the other utility mixins that are built with the same function. It handles errors a little differently — that is, `get-uswds-value` accepts non-USWDS values (like `17px`) with a warning and `units` throws an error. `get-uswds-value()` also "knows" the standard values for padding and will reject something like `neg-2` or `-2` since those negative values are invalid for padding.
